src/Console/Command/ShowCommand.php

```diff
@@ -0,0 +1,507 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Command;
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Console\ConfigurationResolver;
+use PhpCsFixer\Fixer\DeprecatedFixerInterface;
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+use PhpCsFixer\FixerNameValidator;
+use PhpCsFixer\RuleSet;
+use PhpCsFixer\RuleSetInterface;
+use PhpCsFixer\ToolInfoInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Adamo Aerendir Crespi <hello@aerendir.me>
+ * @author Patrick Landolt <landolt@gmail.com>
+ */
+final class ShowCommand extends Command
+{
+    const COMMAND_NAME = 'show';
+    const THICK = "\xE2\x9C\x94";
```

Comment:
This constant and the ones below should be marked `@internal`.

Reply:
Done!